src/main/services/additional-content/local-playlists-manager.service.ts

```diff
@@ -71,10 +72,12 @@ export class LocalPlaylistsManagerService {
             this.windows.openWindow(`oneclick-download-playlist.html?playlistUrl=${bplistUrl}`);
         });
 
-        this.fileAssociation.registerFileAssociation(".bplist", filePath => {
-            log.info("FILE ASSOCIATION RECEIVED", filePath);
-            this.openOneClickDownloadPlaylistWindow(filePath);
-        });
+        for (const filetype of this.PLAYLIST_FILETYPES) {
```

Comment:
I'm not sure if it's a good idea to set `.json` as a file association here 🤔
Doing this would mean that all JSON files on the user's computer would be recognized as openable with BSM to install a playlist

Reply:
Ahhh, so that's what it does. Though I needed to register that as well for file association. Will do a rebase to clean up the PR. Is [9294ca7](https://github.com/Zagrios/bs-manager/pull/621/commits/9294ca7d74ea8d9850376f7c9ed051f751fb8722) expected to have to changes?

Reply:
Or you can make a simple commit to fix that 🤔 PRs will always be squashed before merging now

Reply:
Really, wouldn't a rebase be better though, so it would preserve the all the commits in the future since it appends them at the end of the git history for proper tracking.

Reply:
I've switched it back to a simple merge instead of a squash 😅
But I'll stick with merge instead of rebase since I'm much more familiar with merge (and at least there's no need to force push with a merge 😅)